pyproject.toml

```diff
@@ -24,7 +24,6 @@ no_implicit_reexport = true
 show_column_numbers = true
 show_error_codes = true
 show_error_context = true
-show_none_errors = true
```

Comment:
This is default behavior, and the configuration option is no longer recognized by `mypy`.